packages/components/src/mobile/color-settings/utils.native.js

```diff
@@ -1,8 +1,18 @@
+const getGradientType = ( color ) => {
+	if ( color.includes( 'radial-gradient' ) ) {
+		return 'radial';
+	} else if ( color.includes( 'linear-gradient' ) ) {
+		return 'linear';
+	}
+	return false;
+};
+
 export const colorsUtils = {
 	subsheets: {
 		settings: 'Settings',
 		color: 'Color',
 	},
 	segments: [ 'Solid', 'Gradient' ],
-	isGradient: ( color ) => color?.includes( 'linear-gradient' ),
+	isGradient: !! getGradientType,
```

Comment:
I believe it should be
```suggestion
	isGradient: (color) => !! getGradientType(color),
```